tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -66,14 +66,18 @@ private static string GetAuthority()
             return $"https://login.microsoftonline.com/{tenant}";
         }
 
-        private static IPublicClientApplication CreatePca()
+        private static IPublicClientApplication CreatePca(bool withWamBroker = false)
         {
             var pcaBuilder = PublicClientApplicationBuilder
                             .Create(s_clientIdForPublicApp)
                             .WithAuthority(GetAuthority())
-                            .WithLogging(Log, LogLevel.Verbose, true)
-                            .WithExperimentalFeatures()
-                            .WithDesktopFeatures();
+                            .WithLogging(Log, LogLevel.Verbose, true);
+
+            if(withWamBroker)
+            {
+                pcaBuilder.WithWindowsDesktopFeatures()
+                            .WithBroker();
```

Comment:
Why two calls?

Reply:
WithBroker is required for config. This was there in different file earlier too. [Ref:]https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/da70e8a1bae4ab8ddb1ce659e7ed4c5167aa0334/tests/devapps/NetFxConsoleTestApp/Program.cs#L142)